powerauth-push-server/src/main/java/io/getlime/push/controller/rest/PushCampaignController.java

```diff
@@ -200,19 +200,19 @@ public Response addUsersToCampaign(@PathVariable(value = "id") Long campaignId,
      */
     @RequestMapping(value = "{id}/user/list", method = RequestMethod.GET)
     @ResponseBody
-    public PagedResponse<ListOfUsersFromCampaignResponse> getListOfUsersFromCampaign(@PathVariable(value = "id") Long id, Pageable pageable) {
+    public ListOfUsersPagedResponse<ListOfUsersFromCampaignResponse> getListOfUsersFromCampaign(@PathVariable(value = "id") Long id, Pageable pageable) {
```

Comment:
Why did your client process `ObjectResponse<PagedResponse<ListOfUsersFromCampaignResponse>>` when you server correctly publishes `PagedResponse<ListOfUsersFromCampaignResponse>`?

Reply:
Problem explained above, our private method is processing with ObjectResponse instances.